styles/landing.module.scss

```diff
@@ -1,39 +1,12 @@
-// html {
-//   -webkit-font-smoothing: antialiased;
-//   -moz-osx-font-smoothing: grayscale;
-//   /* font-smoothing: antialiased; */
-//   text-rendering: optimizeLegibility;
-//   font-display: fallback;
-//   background: var(--blue-darkest);
-// }
-
-// body {
-//   margin: 0;
-//   color: #fff;
-// }
-
-// p {
-//   font-family: "proxima_nova", sans-serif;
-//   font-size: 20px;
-//   font-weight: 300;
-//   line-height: 1.4em;
-// }
-
-// h1 {
-//   font-family: "commuters_sans", sans-serif;
-//   font-weight: 300;
-//   font-size: 32px;
-// }
-
-// h2 {
-//   font-family: "commuters_sans", sans-serif;
-//   font-weight: 300;
-//   font-size: 24px;
-//   line-height: 1.4em;
-// }
+@import "styles/mixins.scss";
+@import "styles/variables/colors.module.scss";
 
 .container {
 
+  a:hover {
+    color: $blue-lighter !important;
```

Comment:
is `!important` necessary here? I've had to use it in one place, concerning images, and literally the first time I've had to do so in my professional career. Just want to make sure we explored other options.

Reply:
no, it's just lazy :(  i'll fix it later